### ISSUE TITLE
Need Dummy data for db

### ISSUE THREAD

**Issue by Prashant-Gupta-AI** [LeanKhan/fs-pro#14]
Hi, Is there a dummy data set to poppulate db? I am trying to simulate a game between two teams.

**Comment by LeanKhan** [comment]
@Prashant-Gupta-AI yes it's working, I'm preparing a little writeup and sample data 🙏

**Comment by LeanKhan** [comment]
@Prashant-Gupta-AI hiii, here is sample data. This is a DB dump at a clean slate: https://fspro-cdn.sturves.tech/clean-fspro-dump.zip

And here are directions: https://fspro.sturves.tech/

Please let me know if you need any help, I appreciate your patience and interest 🙏

**Comment by Prashant-Gupta-AI** [comment]
@LeanKhan Hi, thank you for the response. Sorry for the late response but I was stuck in my 9-5 work. I will look into this and let you know.

**Comment by LeanKhan** [comment]
No worries! @Prashant-Gupta-AI
